### DISCUSSION TITLE
Translatable variant names

### DISCUSSION THREAD

**Discussion by freshface** [jonassiewertsen/statamic-butik#190] [Ideas]
It would be nice that the variant name is translatable.

**Comment by jonassiewertsen** [comment]
That will be possible with butik version 4 :-)

**Reply by freshface** [reply]
is there an eta on v4 :) ?

**Reply by jonassiewertsen** [reply]
It's not on the roadmap yet. 

If this is an important feature for you, please open a new discussion to describe your feature request a little more, what it should do etc so others can vote for it. 

If the interest is high enough, I will take a closer look.
